source/_docs/migrate.md

```diff
@@ -6,107 +6,64 @@ tags: [migrate, getting-started]
 keywords: migrate, migrating site, migrate from remote host, migrate existing site, migrate from other host, migrate from another host, how to migrate an existing site, alternate host, another host, migration, migrations, migrates, move site to pantheon, move from remote host, move from current host, move hosts, changing hosting providers, how to move hosting to pantheon
 ---
 
-Your site migration has four phases. You’ll package your site, import it, test it out, and then change DNS and go live. With a good plan and understanding of the platform, the process will run smoothly.
+Your site migration has four phases. You’ll package your site, import it, test it out, and then [go live](/docs/going-live) by adding your domain and changing DNS.
 
-We recommend migrating WordPress sites from another host using the [Pantheon Migration](https://wordpress.org/plugins/bv-pantheon-migration/) plugin, developed by [BlogVault](https://blogvault.net/). For more details, see [Migrate to Pantheon: WordPress](/docs/migrate-wordpress).
-
-## Create Archives
-
-In this phase, you will create an archive of your site.
-
-Are you running another application in addition to WordPress or Drupal? You need to remove it. See [Platform Considerations](/docs/platform-considerations/#one-application-per-site).
-
-You’ll need to package up your:
-
-- **Codebase** - All executable code, including core, custom and contrib modules or plugins, themes, and libraries. For the suggested directory listing of your site’s codebase, see our [Drupal](/docs/drupal-export#manually-create-archive) or [WordPress](/docs/wordpress-export#manually-create-separate-site-archives) export documentation.
-
-- **Database** - A single .sql dump, contains the content and active state of the site's configurations.
+<div class="alert alert-info" role="alert">
+<h4>Note</h4>
+Each site supports a single Drupal or WordPress application. Placing a WordPress application to behave as the blog for a Drupal site, for example, is unsupported.
+</div>
+## Migrating WordPress Sites
+Follow the recommended process within [Migrate to Pantheon: WordPress](/docs/migrate-wordpress), which uses the Pantheon Migration plugin.
 
-- **Files** - Anything in `sites/default/files` for Drupal or `wp-content/uploads` for WordPress. This houses a combination of uploaded content from site users, along with generated stylesheets, aggregated scripts, image styles, etc.
+The following scenarios are exceptions to the recommended process and require [manually migrating](https://pantheon.io/docs/manual-import/) the site:
 
+- Your site requires a custom upstream.
+- You would like to preserve the site's existing Git history.
+- [WordPress Site Networks](/docs/wordpress-site-networks)
```

Comment:
one more scenario:
- You can't install a plugin on your existing site (e.g. wordpress.com)